combine_rewards.py

```diff
@@ -1,4 +1,10 @@
+import decimal
+import json
+
+from decimal import Decimal
```

Comment:
Importing decimal.Decimal and Decimal concurrently?

Reply:
I was originally using `decimal.?` for something else but it got nuked -- Removed this.